pom.xml

```diff
@@ -156,6 +162,18 @@
       <artifactId>guava</artifactId>
       <version>19.0-rc2</version>
     </dependency>
+    
+    <dependency>
+  		<groupId>org.eclipse.jetty</groupId>
```

Comment:
fix XML indentation please,
add variable as in ${LUCENE_VERSION} for Jetty so you can reference to versions consistently in one place.